### DISCUSSION TITLE
Minor issue: URL Input

### DISCUSSION THREAD

**Discussion by T-prog3** [TumblThreeApp/TumblThree#279] [Ideas]
I usually doesn't add blogs this way, but shouldn't the `Enter URL:` input bars value be cleared if the user presses `Add Blog` and there's no error?

**Comment by thomas694** [comment]
That depends. Sometimes I found it convenient to only change part of the URL for adding another blog.
Since a message is displayed, when (accidentally) clicking `Add Blog` again, there was no hurry to clear the field.
